test/tools/Modules/WebListener/WebListener.psm1

```diff
@@ -18,6 +19,76 @@ Class WebListener
 
 [WebListener]$WebListener
 
+function New-RandomHexString
+{
+    param([int]$Length = 10)
+
+    $random = [Random]::new()
+
+    return ((1..$Length).ForEach{ '{0:x}' -f $random.Next(0xf) }) -join ''
```

Comment:
This code is the same as `New-CertificatePassword` can we re-use the code?

Reply:
@rjmholt How about in here: https://github.com/PowerShell/PowerShell/blob/master/test/tools/Modules/HelpersCommon/HelpersCommon.psm1